modules/apps/wiki/wiki-service/src/main/java/com/liferay/wiki/service/impl/WikiPageServiceImpl.java

```diff
@@ -549,6 +549,18 @@ public WikiPage getPageByPageId(long pageId) throws PortalException {
 		return page;
 	}
 
+	@Override
+	public WikiPage getPageByResourcePrimKey(long resourcePrimKey)
```

Comment:
getPageByResourcePrimKey > getWikiPage...

it return a wikiPage ... :D

Reply:
Oh!, I see that on that class ALL the methods have the same name...
Maybe is better to maintain the same as other methods